packages/invoice-dashboard/src/utils/generateInvoice.ts

```diff
@@ -0,0 +1,178 @@
+import { formatUnits } from "viem";
+import { loadScript } from "./loadScript";
+import { calculateItemTotal } from "@requestnetwork/shared-utils/invoiceTotals";
+
+declare global {
+  interface Window {
+    html2pdf: any;
+  }
+}
+
+async function ensureHtml2PdfLoaded() {
+  if (typeof window.html2pdf === "undefined") {
+    try {
+      await loadScript(
+        "https://cdnjs.cloudflare.com/ajax/libs/html2pdf.js/0.10.1/html2pdf.bundle.min.js"
+      );
+    } catch (error) {
+      console.error("Failed to load html2pdf script:", error);
+    }
+  }
+}
```

Comment:
After reviewing the entire PR, I've realized that CodeRabbit went way too far in its suggestions to catch and log errors in internal utilities.

Errors should be handled at the top level, not in internal utilities. Internal utilities should pass the error up and allow the top level to handle them.

@sstefdev Please revert the `try/catch` blocks you added to internal utilities. Keep the `try/catch` block you added to top-level components.

Reply:
Resolved by adding Shadcui and Toaster, and also removing low level `try/catch` blocks